CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [6.1.3](https://github.com/cleandart/react-dart/compare/6.1.2...6.1.3)
+
+- [#308] Bump elliptic from 6.5.3 to 6.5.4
+- [#311] Bump ssri from 6.0.1 to 6.0.2
+- [#312] Bump lodash from 4.17.20 to 4.17.21
+- [#313] Bump browserslist from 4.16.1 to 4.16.6
+- [#318] Bump path-parse from 1.0.6 to 1.0.7
```

Comment:
These dependencies got updated but it looks like we never updated the bundle...

It looks like we don't have a CI check that builds the bundle and verifies it's up to date 😕 

Can we update the bundle in this PR?

Reply:
@greglittlefield-wf when I ran `yarn build` locally, there were no changes.